### PR TITLE
Correctly reset fResult before final checks

### PR DESCRIPTION
**Pull request by kwek20**
We check for false everywhere else, and `__leave` when it is false. Thus returning `fResult = false`.
However in the end when `CertVerifyCertificateChainPolicy` returns `true`, and thus sets fResult to `true`, but then any of the checks on line 452...473 fail, we still return fResult in the end. 

Can alternatively be fixed by making another BOOL variable and just setting that to false by default, and  TRUE on the end of the `try`, and returning that in the end 🤷 

tl:dr
Set false before final checks to prevent `true` returned on a valid signed cert, but not the one we wanted

### CODE REVIEW

**Comment by rajivshah3** [comment]
Thanks Brord!
